### DISCUSSION TITLE
Has `README.rst` grown too much?

### DISCUSSION THREAD

**Discussion by vytas7** [falconry/falcon#2086] [Polls]
Our [`README.rst`](https://github.com/falconry/falcon/blob/master/README.rst) has grown "organically" with time, and it has become rather lengthy, challenging the average hipster's attention span. The same `README` is used on our PyPi project page.

It now features lengthy installation instructions, introductory code samples, where the "more complex example" is relatively elaborate, and it is presented in two incarnations -- WSGI & ASGI, etc.

**Comment by CaselIT** [comment]
Maybe the "more complex example" could be transformed to a link to the doc?
